Makefile

```diff
@@ -481,7 +481,8 @@ build-ci:
 # - node-test-commit-linux-coverage: where the build and the tests need
 #   to be instrumented, see `coverage`.
 run-ci: build-ci
-	$(MAKE) test-ci
+	# All the sub target in `test-ci` already have internal parallelism.
```

Comment:
Maybe move this comment above the target as currently it will be in the build logs:
```
12:44:42 make[1]: Leaving directory `/data/iojs/build/workspace/node-test-commit-linuxone/nodes/rhel72-s390x'
12:44:42 # All the sub target in `test-ci` already have internal parallelism.
12:44:42 make test-ci -j1
```

Reply:
arrr `make` and it's idiosyncrasies